R/pkg/R/functions.R

```diff
@@ -488,19 +488,35 @@ setMethod("initcap",
             column(jc)
           })
 
-#' isNaN
+#' isnan
 #'
-#' Return true iff the column is NaN.
+#' Return true if the column is NaN.
 #'
-#' @rdname isNaN
-#' @name isNaN
+#' @rdname isnan
+#' @name isnan
 #' @family normal_funcs
 #' @export
-#' @examples \dontrun{isNaN(df$c)}
-setMethod("isNaN",
+#' @examples \dontrun{isnan(df$c)}
+setMethod("isnan",
           signature(x = "Column"),
           function(x) {
-            jc <- callJStatic("org.apache.spark.sql.functions", "isNaN", x@jc)
+            jc <- callJStatic("org.apache.spark.sql.functions", "isnan", x@jc)
+            column(jc)
+          })
+
+#' isnull
+#'
+#' Return true if the column is NULL.
+#'
+#' @rdname isnull
+#' @name isnull
+#' @family normal_funcs
+#' @export
+#' @examples \dontrun{isnull(df$c)}
+setMethod("isnull",
```

Comment:
Our replies crossed - it would seem to me to say Scala/Python `isnull` == R `is.na` is confusing.

In any case we could definitely use some explanations on NULL/NA in the SPARKR programming guide, opened JIRA https://issues.apache.org/jira/browse/SPARK-12071